gcloud-java-storage/src/main/java/com/google/gcloud/storage/CopyWriter.java

```diff
@@ -32,7 +32,13 @@
 import java.util.concurrent.Callable;
 
 /**
- * Google Storage blob copy writer. This class holds the result of a copy request. If source and
+ * Google Storage blob copy writer. A {@code CopyWriter} object allows to copy both blob's data and
+ * information. To override source blob's information call {@link Storage#copy(Storage.CopyRequest)}
```

Comment:
Fine as is. I would just have said something like "To override source blob's information supply a BlobInfo as the target" or something like but if you think what you have is more clear feel free to keep it.